tests/source/disable_all_formatting.rs

```diff
@@ -0,0 +1,4 @@
+// rustfmt-disable_all_formatting: true
```

Comment:
You're right. I misread the Contributing.md instructions and put the file in the wrong directory.

> Creating a test is as easy as creating a new file in ./tests/source/ and an equally named one in ./tests/target/. If it is only required that rustfmt leaves a piece of code unformatted, it may suffice to only create a target file.